packages/react-components/src/components/Picker/TriggerBody.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+import { IPickerListItem } from './PickerList';
+import { Tag } from '../Tag';
+import { PickerType } from './Picker';
+
+import styles from './TriggerBody.module.scss';
+
+const baseClass = 'picker-trigger-body';
+
+interface ITriggerBodyProps {
+  items: IPickerListItem[];
+  type: PickerType;
+  onItemRemove: (item: IPickerListItem) => void;
+}
+
+export const TriggerBody: React.FC<ITriggerBodyProps> = ({
+  items,
+  type,
+  onItemRemove,
+}) => {
+  if (type === 'single') {
```

Comment:
isn't better to have 2 separate components, one for `single picker` and second one for `multiple picker`?